test/orm/declarative/test_tm_future_annotations.py

```diff
@@ -293,11 +290,11 @@ class MyClass(decl_base):
             data: Mapped[str] = mapped_column(default="some default")
 
             if reference_type.container_w_remote_mapped:
-                status: ClassVar[Dict[str, MyOtherClass]]
+                status: ClassVar[dict[str, MyOtherClass]]
             elif reference_type.container_w_local_mapped:
-                status: ClassVar[Dict[str, MyClass]]
+                status: ClassVar[dict[str, MyClass]]
             elif reference_type.plain_optional:
-                status: ClassVar[Optional[int]]
```

Comment:
these are tests for parsing of the `Optional[]` construct.  These can't be changed